### ISSUE TITLE
implement facebook domainbed mnist neural network 

### ISSUE THREAD

**Issue by smilesun** [marrlab/DomainLab#620]
currently our mnist network has batchnormalization, the domainbed network does not have this
